Sequential.lua

```diff
@@ -23,6 +23,16 @@ function Sequential:insert(module, index)
    self.gradInput = self.modules[1].gradInput
 end
 
+function Sequential:remove(index)
+   index = index or #self.modules
+   if index > #self.modules or index < 1 then
+      error"index out of range"
+   end
+   table.remove(self.modules, index)
+   self.output = self.modules[#self.modules].output
```

Comment:
Don't we need to make the input of the (old) `index+1` module be the output of the `index-1` module as well ? Same thing for the gradInput.